app/views/courses/show.html.erb

```diff
@@ -0,0 +1,61 @@
+<%= csrf_meta_tag %>
+<script> 
+// so we can access the correct urls in the .js files
+var flashcard_url = "<%= url_for(controller:'courses', action:'flashcard') %>";
+var note_upload_url = "<%= url_for(controller:'courses', action:'updateNotes') %>";
+</script>
+
+<%= javascript_include_tag "dropzone.js" %>
+
+<center>
+  <br>
+    <div>
+      <h1> <%= @course.code %>: <%= @course.title %></h1>
+      <ul class="nav nav-pills nav-justified">
+        <li class = "active"><%= link_to "Course", user_course_path(@user, @course) %><br></li>
+        <li><%= link_to "Edit Course", edit_user_course_path(@user, @course) %><br></li>
+        <li><%= link_to "Student Roster", user_course_students_path(@user, @course) %><br></li>
+        <li><%= link_to "Face/Name Quiz", {controller:"courses", action:"quiz"} %><br></li>
+      </ul>
+      <div class = "container">
+        <div class="row">
+          <div class = "col-lg-4"></div>
```

Comment:
Is this div empty on purpose? If not, is it worth replacing it with a col-offset on the next div?

Reply:
I think that might have been left over from something one of use deleted but didn't get rid of the tags. Something to clean up.